spec/support/haiku_helper.rb

```diff
@@ -8,6 +8,16 @@ def submit_haiku(poem = 'A haiku', by: nil)
     fill_in 'Your haiku', with: poem
     click_on 'Send'
   end
+
+  def approve_haiku(poem, by: nil)
+    # TEMP this should eventually use admin approve view:
+    submissions = Submission.where(body: poem)
+    submissions = submissions.where(submitted_by: by) if by.present?
+
+    return unless submissions.one?
```

Comment:
Should this raise an error? If I wrote this test:

```ruby
submit_haiku('A haiku!')
submit_haiku('A haiku!')
approve_haiku('A haiku!')
```

I would not expect _neither_ haiku to be approved.

How about…

```suggestion
fail("Could not find one haiku matching approval criteria") unless submissions.one?
```